src/common/Smi.Common/Messages/Updating/UpdateValuesMessage.cs

```diff
@@ -70,37 +68,5 @@ public override string ToString()
             return 
                 $"{nameof(UpdateValuesMessage)}: {nameof(WhereFields)}={string.Join(",",WhereFields)} {nameof(WriteIntoFields)}={string.Join(",",WriteIntoFields)}";
         }
-
-        /// <summary>
-        /// Checks whether two messages update the same fields using the same where logic
-        /// </summary>
-        /// <param name="obj"></param>
-        /// <returns></returns>
-        public override bool Equals(object obj)
-        {
-            return obj is UpdateValuesMessage message &&
-                   Enumerable.SequenceEqual(Operators ?? new string[0], message.Operators?? new string[0]) &&
```

Comment:
Does `MemberwiseEquatable<UpdateValuesMessage>` handle array equality transparently using `Enumerable.SequenceEqual`?

Reply:
It does yeah, see [here](https://github.com/thedmi/Equ#equ). I've also added `tests/common/Smi.Common.Tests/EquTests.cs` which we can use to sanity-check this ourselves.